php-cs-test/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM composer
+
+RUN composer global require prooph/php-cs-fixer-config
+
+COPY rules.php /rules.php
+
+VOLUME /sut
+WORKDIR /sut
+
+ENTRYPOINT ["/composer/vendor/bin/php-cs-fixer", "fix", "--dry-run", "--diff", "--config=/rules.php"]
```

Comment:
@bweston92 What do you think about these lines, so the CMD arguments can be overridden?

```
CMD ["fix", "--dry-run", "--diff"]
ENTRYPOINT ["/composer/vendor/bin/php-cs-fixer", "--config=/rules.php"]
```

The composer path is another path with `prooph/composer`.